caffe2/experiments/operators/fully_connected_op_decomposition.h

```diff
@@ -67,7 +67,7 @@ class FullyConnectedOpDecomp final : public Operator<Context> {
     std::vector<int64_t> dims;
     if (X.dim() > 1) {
       dims = {M, N};
-      multi_buffer_.Resize(M, middle);
+      ReinitializeTensor(&multi_buffer_, {M, middle}, at::dtype<T>().device(Context::GetDeviceType()));
```

Comment:
`.device(Context::GetDeviceType()` is unnecessary right?

Reply:
`ReinitializeTensor` is a free function, we need to provide device